src/Faker/Provider/az_AZ/Person.php

```diff
@@ -0,0 +1,169 @@
+<?php
+
+namespace Faker\Provider\az_AZ;
+
+class Person extends \Faker\Provider\Person
+{
+    protected static $maleNameFormats = array(
+        '{{firstNameMale}} {{middleNameMale}} {{lastName}}',
+        '{{lastName}} {{firstNameMale}} {{middleNameMale}}',
+    );
+
+    /**
+     * This provider uses wikipedia's 250 top russian last names
+     * That list of MALE last names could be safely extended to FEMALE list just by adding 'a' letter at the end
+     */
+    protected static $femaleNameFormats = array(
+        '{{firstNameFemale}} {{middleNameFemale}} {{lastName}}а',
+        '{{lastName}}а {{firstNameFemale}} {{middleNameFemale}}',
+    );
+
+    /**
+     * {@link} http://www.azerbaijans.com/content_597_az.html
```

Comment:
content from this site is copyrighted, please find an open-source source or remove.